src/trie.rs

```diff
@@ -247,7 +310,11 @@ impl Trie {
 
 impl Statistics for Trie {
     fn heap_bytes(&self) -> usize {
-        self.mapper.heap_bytes() + self.nodes.len() * core::mem::size_of::<Node>()
+        self.mapper.heap_bytes() + self.nodes.len() * size_of::<Node>()
```

Comment:
ditto

Reply:
@kampersanda reminder

Reply:
@vbkaisetsu 
I've already fixed it, but Files Changed is not updated...
See https://github.com/daac-tools/crawdad/blob/add-serialize/src/trie.rs#L313

Reply:
Ah... I commented to heap_bytes(). It's my mistake.